Test/Utils/ContrailAPI/VirtualRouter.ps1

```
@@ -0,0 +1,52 @@
+. $PSScriptRoot\Constants.ps1
+
+function Get-ContrailVirtualRouterUuidByName {
+    Param ([Parameter(Mandatory = $true)] [string] $ContrailUrl,
+           [Parameter(Mandatory = $true)] [string] $AuthToken,
+           [Parameter(Mandatory = $true)] [string] $RouterName)
+
+    $RequestUrl = $ContrailUrl + "/virtual-routers"
+    $Response = Invoke-RestMethod -Uri $RequestUrl -Headers @{"X-Auth-Token" = $AuthToken} -Method Get
+    $Routers = $Response.'virtual-routers'
+
+    foreach ($Router in $Routers) {
+        $FqName = $Router.fq_name
+        $AreFqNamesEqual = $null -eq $(Compare-Object $FqName @("default-global-system-config", $RouterName))
+
+        if ($AreFqNamesEqual) {
+            return $Router.uuid
+        }
+    }
+
+    return $null
```

Change suggestion: can we throw an exception here?